python/pyarrow/array.pxi

```diff
@@ -750,6 +750,28 @@ cdef class _PandasConvertible(_Weakrefable):
         Returns
         -------
         pandas.Series or pandas.DataFrame depending on type of object
+
+        Examples
+        --------
+        Convert a RecordBatch to pandas DataFrame:
+
+        >>> import pyarrow as pa
+        >>> n_legs = pa.array([2, 4, 5, 100])
+        >>> animals = pa.array(["Flamingo", "Horse", "Brittle stars", "Centipede"])
+        >>> batch = pa.RecordBatch.from_arrays([n_legs, animals],
+        ...                                     names=["n_legs", "animals"])
```

Comment:
I'm wondering if it would be better to use the `pa.record_batch` constructor since that more closely resembles the syntax of `pd.DataFrame` that Pandas users are familiar with.
```suggestion
        >>> batch = pa.record_batch({"n_legs": nlegs, "animals": animals})
```

Reply:
Yes, good point! Corrected.
Thanks for the reviews!